### ISSUE TITLE
Saving images doesn't work

### ISSUE THREAD

**Issue by graue** [jeena/Bungloo#115]
On Linux, when I right click a photo and do Save Image, it doesn't do anything.

(Also, not sure the other stuff in the menu does what it's supposed to either... for example "Copy Image Address" copies a humongous data: URL to the clipboard which makes Firefox freeze for a few seconds if I paste it into the address bar.)

**Comment by jeena** [comment]
Same thing happens on OS X too. The big data URL is what is used within of Tentia to show the photo.

The other stuff in the menu is also just default stuff which I never looked at actually, will need to remove the unnecessary things.

**Comment by poweruser82** [comment]
Don't you use 
GET /posts/:post_id/attachments/:name

to get the image?

Inviato da iPhone

Il giorno 08/gen/2013, alle ore 21:46, Jeena Paradies notifications@github.com ha scritto:

**Comment by jeena** [comment]
Yes I do and then I put it into a data-URI and into <img src=""> The thing is that I have to do it with a XmlHttpRequest so I can send the authentication headers and stuff if I only would put the url into the <img src="url"> then I wouldn't be able to do that.

**Comment by poweruser82** [comment]
Can't you store the attachment on disk and then use a file:// URL or somewhere in the cache?

Inviato da iPhone

Il giorno 09/gen/2013, alle ore 10:36, Jeena Paradies notifications@github.com ha scritto:

**Comment by jeena** [comment]
Hm yeah, that is a good idea, I will try to do that :-)

**Comment by poweruser82** [comment]
Temporary solution on OS X. Copy the image via the contextual menu, and make a new document in Previw.app
